src/coreclr/ilasm/asmparse.y

```diff
@@ -96,7 +96,7 @@
 %token _CLASS _NAMESPACE _METHOD _FIELD _DATA _THIS _BASE _NESTER
 %token _EMITBYTE _TRY _MAXSTACK _LOCALS _ENTRYPOINT _ZEROINIT
 %token _EVENT _ADDON _REMOVEON _FIRE _OTHER
-%token _PROPERTY _SET _GET DEFAULT_
```

Comment:
This has been defined multiple times, so remove it here.

Reply:
Yup, makes sense.